docs/repositories.rst

```diff
@@ -26,20 +26,26 @@ Bug tracker is in GitHub and can be used for almost anything add-ons related. `E
 
 addons-server
 ~~~~~~~~~~~~~
-The `addons.mozilla.org <https://addons.mozilla.org>`__ website. The `repository <https://github.com/mozilla/addons-server>`_ is on GitHub. The preferred bug tracker is `github <https://GitHub.com/mozilla/addons-server/issues/>`__ although components do exist on bugzilla for legacy bugs. Documentation is `on readthedocs <https://addons-server.readthedocs.io>`__.
+The `addons.mozilla.org <https://addons.mozilla.org/>`__ website. The `repository <https://github.com/mozilla/addons-server>`__ and `issue tracker <https://github.com/mozilla/addons-server/issues/>`__ is on GitHub. Documentation is `on readthedocs <https://addons-server.readthedocs.io>`__.
 
 In the past this repository has been known as *remora*, *zamboni* or *olympia*.
 
-amo-validator
-~~~~~~~~~~~~~
-The legacy Python version of the validator for the `addons.mozilla.org <https://addons.mozilla.org>`__ website. The `repository <https://github.com/mozilla/amo-validator>`__ is on GitHub. The preferred bug tracker is `GitHub <https://github.com/mozilla/amo-validator/issues/>`__ although components do exist on bugzilla for legacy bugs. Documentation is `on readthedocs <https://addons-server.readthedocs.io>`__.
+addons-code-manager
+~~~~~~~~~~~~~~~~~~~
+A web application to manage add-on source code, such as reviewing code for add-ons submitted to addons.mozilla.org. The `repository <https://github.com/mozilla/addons-code-manager>`__ and `issue tracker <https://github.com/mozilla/addons-code-manager/issues/>`__ is on GitHub.
 
 addons-linter
 ~~~~~~~~~~~~~
-The new JavaScript version of the validator for the `addons.mozilla.org <https://addons.mozilla.org>`__ website. The `repository <https://github.com/mozilla/addons-linter>`__ is on GitHub. The bug tracker is `GitHub <https://github.com/mozilla/addons-linter/issues/>`__. Documentation is `on readthedocs <https://addons-server.readthedocs.io>`__. See also `dispensary <https://github.com/mozilla/dispensary>`__.
+The linter is linting WebExtensions for `addons.mozilla.org <https://addons.mozilla.org/>`__ and `web-ext <https://github.com/mozilla/web-ext/>`__. The `repository <https://github.com/mozilla/addons-linter>`__, `issue tracker <https://github.com/mozilla/addons-linter/issues/>`__ and `documentation <https://mozilla.github.io/addons-linter/>`__ is on GitHub.
```

Comment:
I wonder if `linting` should be expanded into a description of what that means in this case?

Reply:
Do you have a specific copy in mind?

Reply:
I was thinking something like "The linter checks web extensions for common errors and potential problems".

Reply:
Thank you, added in f2a597a.